package.json

```diff
@@ -221,6 +230,8 @@
 		"lint-css:fix": "npm run lint-css -- --fix",
 		"lint:md-js": "wp-scripts lint-md-js",
 		"lint:md-docs": "wp-scripts lint-md-docs",
+		"native": "npm run --prefix packages/react-native-editor",
+		"react-native": "react-native --root packages/react-native-editor",
```

Comment:
What's the difference between those two scripts?

Reply:
The first one invokes the commands that come after and that are in `packages/react-native-editor/package.json`
The second one invokes the [react-native-cli](https://github.com/react-native-community/cli) command.
Thinking about it now, I don't see a need for the `react-native` command here, so I'll remove it.